readme.html

```diff
@@ -290,3 +290,17 @@ <h2>Port Information</h2>
       </tr>
     </table>
   </p>
+  <h2>Playbook Backward Compatibility</h2>
```

Comment:
the indentation for this new block looks inconsistent

Reply:
made consistent indentation for the new block.